index.rst

```diff
@@ -1,11 +1,9 @@
-
-
-
-
 :tocdepth: 1
 
 .. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
+**DOI:** `10.5281/zenodo.192828 <https://doi.org/10.5281/zenodo.1492936>`_
```

Comment:
That looks like a typo in the DOI text above, though the link is correct: 10.5281/zenodo.1492936